codes/classical/q-ary_digits/goppa_codes.yml

```diff
@@ -0,0 +1,57 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: goppa
+physical: q-ary_digits
+logical: q-ary_digits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Goppa Code'
+
+# First reference with the code (optional).
+introduced: '\url{https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=1055088} \(not original\)'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Let G(z) be a Goppa polynomial with coefficients from GF\(q^m\) for some fixed integer m. Let L be a finite subset of the extension field GF(q^m) where q is prime, meaning L  = \{\alpha_1, \cdots, \alpha_n\} is a subset of GF\(q^m\) such that \alpha_i \neq 0 for all \alpha_i \in L. A Goppa code \Gamma\(L,G\) is an [n,k,d] linear code consisting of all vectors a = \(a_1, \cdots, a_n\) 
```

Comment:
Enclose math symbols like G(z), GF(q), q, greek letters etc. entirely in \\( and \\).

Reply:
I think we can avoid using "Goppa polynomial" terminology? Concise encyclopedia of coding theory pg. 322 says that one just needs a polynomial that isn't zero at any of the alphas.

Reply:
"is a subset of GF\(q^m\) such that \alpha_i \neq 0 for all \alpha_i \in L" --> "is a subset of nonzero elements of \(GF(q^m)\)."